### ISSUE TITLE
Add possibility to exclude particular files from processing

### ISSUE THREAD

**Issue by smashercosmo** [css-modules/css-modules-require-hook#43]

**Comment by sullenor** [comment]
I wonder if you can provide any more information about why do you want to do that? :)

**Comment by axelson** [comment]
+1

Would be nice for integrating with external libraries that expect very specific class names

**Comment by sullenor** [comment]
I published a beta version with `ignore` option, hope that helps: [3.0.0-beta](https://github.com/css-modules/css-modules-require-hook/releases/tag/3.0.0-beta)

I will publish stable version a bit later, still need to update documentation and demo example.

**Comment by sullenor** [comment]
Should be available with version [3.0.0](https://github.com/css-modules/css-modules-require-hook/releases/tag/3.0.0)

**Comment by PeteJodo** [comment]
@sullenor I don't know if it's intentional but [this bit of code](https://github.com/PeteJodo/css-modules-require-hook/blob/master/lib/attachHook.js#L12-L14) attempts to call `undefined` as a function.
```js
if (isException(filename)) {
    existingHook(m, filename);
}
```
I think should look more like:
```js
if (isException(filename)) {
    if (existingHook) {
        existingHook(m, filename);
    }
}
```

Now I can see the reason why you'd maybe want to call the function anyway because it means that the user forgot to set an import handler for that file they're ignoring. What would the effects be if `existingHook` doesn't get called?
